plugins/modules/datadog_monitor.py

```diff
@@ -368,6 +394,9 @@ def install_monitor(module):
         "new_host_delay": module.params['new_host_delay'],
         "evaluation_delay": module.params['evaluation_delay'],
         "include_tags": module.params['include_tags'],
+        "notification_preset_name": module.params['notification_preset_name'],
+        "renotify_occurrences": module.params['renotify_occurrences'],
+        "renotify_statuses": module.params['renotify_statuses'],
```

Comment:
Are these kwargs supported by every version of `datadogpy` that the module supports? If not, they must only be passed if the version of `datadogpy` supports them (and the module needs to error out if they have values, and are not supported).

Also does adding these new options change idempotency for existing module usages (which might have these new values set through other means) when these new values are not specified?

Reply:
I checked the `datadogpy` library and it looks like it does not evaluate the parameters. It is based on `datadog-api-client-python`, so if `datadogpy` uses the latest version of `datadog-api-client-python` everything should be good.

Regarding the idempotency: I tested it; when I do not specify the values, it sets `null` and API will use the default for that option.

Reply:
Hmm, but if now the module changes the values of these options to the defaults when they are not specified to the module, this would be a breaking change, as the previous version of the module would not modify these options at all.

Reply:
As I understand this, the previous version of module is sending the call with all the parameters it has. When API sees it doesn't have, for example, `notification_preset_name`, it treats it as a `null` so it just uses the default option for it. In the new version, you send all the parameters, and if you do not specify it in the ansible, it sends `null`, so API will use the same default option as previously because it's still `null`.